### PR TITLE
Smart society services removed

### PR DESCRIPTION
**Pull request by Sander3003**
Smart society services is out of business

### CODE REVIEW

**Comment by CLAAdmin** [comment]
[CLA assistant check](http://cla.opensmartgridplatform.org:80/OSGP/Documentation?pullRequest=208) <br/>You should sign our Contributor License Agreement in order to get your pull request merged.<br/>

**Comment by Sander3003** [comment]
@phoutman is happy with the changes, it can be merged if maintaier @rlemmers agrees.

File: Support/README.md [hunk 1]
```diff
@@ -8,9 +8,5 @@ Community members can help you on voluntary basis. See the open source and commu
 ## Commercial support
 If you'd like professional support for your Open Smart Grid Platform use case, consider support by one of the companies below.
```
Comment (rlemmers) [inline Support/README.md]: In the new version, there are no companies at all in the "Commercial support" section. Can an interested party contact Alliander (and provide contact details)? If so, mention this. If not, mention explicitly there's currently no commercial support.

Comment (Sander3003) [reply Support/README.md]: info@opensmartgridplatform.org would be the best way. Alliander is currenty the company answering the e-mails. This cloud change later on. I will make this clear in the text. Thanks for your input.

Comment (Sander3003) [reply Support/README.md]: Is /home/dev/Sources/SmartSocietyServices/Configuration/ based on a Vagrant script or example installation?

**Comment by Sander3003** [comment]
Is /home/dev/Sources/SmartSocietyServices/Configuration/ based on a Vagrant script or example installation? If so, it could be changed with the new directory. If it's a non-related example, I can change it to osgp

**Comment by rlemmers** [comment]
Good question, @Sander3003. The location /home/dev/Sources/SmartSocietyServices/Configuration/ is created and used by the Vagrant / rollout scripts. Because we aren't changing those scripts (yet), it's better to leave that .md file as it is.

File: Support/README.md [hunk 2]
```diff
@@ -6,11 +6,4 @@ There are multiple options for support
...
 ## Commercial support
-If you'd like professional support for your Open Smart Grid Platform use case, consider support by one of the companies below.
-
-Smart Society Services
-- Website: http://smartsocietyservices.com
-- Contact: http://smartsocietyservices.com/contact/
-
-Your company here? Contact us!
-- http://opensmartgridplatform.org/contact/
+Their is currently no commercial support availible. If you'd like to provide professional support, consider starting a support business/company and metion your company name here (pull-request or info@opensmartgridplatform.org).
```
Comment (rlemmers) [inline Support/README.md]: My proposal: "Currently there's no commercial support available. If you would like to provide commercial support, contact us and we'll add your company name here."

If you don't like my proposal, you can just correct the spelling mistakes: "Their => There" and "metion" => "mention".

Comment (Sander3003) [reply Support/README.md]: I used your proposal.

**Pull request merged**